pkg/tsdb/cloudwatch/query_transformer.go

```diff
@@ -104,10 +99,119 @@ func (e *cloudWatchExecutor) transformQueryResponsesToQueryResult(cloudwatchResp
 			queryResult.ErrorString = "Cloudwatch GetMetricData error: Too many datapoints requested - your search has been limited. Please try to reduce the time range"
 		}
 
+		eq, err := json.Marshal(executedQueries)
+		if err != nil {
+			plog.Error("Could not marshal executedString struct", err)
+		}
+
+		link, err := buildDeepLink(refID, requestQueries, executedQueries, startTime, endTime)
+		if err != nil {
+			plog.Error("Could not build deep link", err)
+		}
+
+		for _, frame := range frames {
+			frame.Meta = &data.FrameMeta{
+				ExecutedQueryString: string(eq),
+			}
+
+			if link != "" {
+				for _, field := range frame.Fields {
+					field.Config = &data.FieldConfig{
+						Links: []data.DataLink{
+							{
+								Title:       "View in CloudWatch console",
+								TargetBlank: true,
+								URL:         link,
+							},
+						},
+					}
+				}
+			}
```

Comment:
It's more idiomatic to avoid deep nesting in Go.
```suggestion
			if link != "" {
				continue
			}
			
				for _, field := range frame.Fields {
					field.Config = &data.FieldConfig{
						Links: []data.DataLink{
							{
								Title:       "View in CloudWatch console",
								TargetBlank: true,
								URL:         link,
							},
						},
					}
				}
```

Reply:
Yes, good idea!